packages/financial-templates-lib/test/clients/InsuredBridgeL2Client.js

```diff
@@ -34,6 +34,24 @@ const instantRelayFeePct = toWei("0.005");
 const quoteTimestampOffset = 60; // 60 seconds into the past.
 
 describe("InsuredBridgeL2Client", () => {
+  const generateDepositHash = (depositData) => {
```

Comment:
Can you not re-use the L2Client's `generateDepositHash` function?

Reply:
we could but it's a bit janky to use the thing you are testing in the test 🤔

Reply:
true. Can ignore this thread now.